main.go

```diff
@@ -225,30 +226,26 @@ func convertField(curPkg *ProtoPackage, desc *descriptor.FieldDescriptorProto, m
 
 	opts := desc.GetOptions()
 	if opts != nil && proto.HasExtension(opts, protos.E_Bigquery) {
-		rawOpt, err := proto.GetExtension(opts, protos.E_Bigquery)
-		if err != nil {
-			return nil, err
-		}
-		opt := *rawOpt.(*protos.BigQueryFieldOptions)
-		if opt.Ignore {
+		opt := proto.GetExtension(opts, protos.E_Bigquery).(*protos.BigQueryFieldOptions)
+		if opt.GetIgnore() {
 			// skip the field below
 			return nil, nil
 		}
 
-		if opt.Require {
+		if opt.GetRequire() {
 			field.Mode = "REQUIRED"
 		}
 
-		if len(opt.TypeOverride) > 0 {
-			field.Type = opt.TypeOverride
+		if to := opt.GetTypeOverride(); len(to) > 0 {
```

Comment:
There is a well-known abbreviation for “type”:
```suggestion
		if typ := opt.GetTypeOverride(); typ != "" {
```
Prefer `!= ""` or `== ""` when we semantically want to be testing against it being the empty string. (A good compiler should be able to optimize this to `len(myString) != 0`) And use `len(myString) > index` only when one is about to index into the string.

Reply:
It was supposed to be `to` for type override, and the len was already there, but I'll change it.

Reply:
There used to be some pretty common premature optimization. :)

Reply:
Done